### ISSUE TITLE
rename Axis.translate to Axis.index

### ISSUE THREAD

**Issue by gdementen** [liam2/larray#479]

**Comment by gdementen** [comment]
or maybe keep both and have .index for single label and .translate for single labels + all other stuff (slices, groups, whatever).

**Comment by alixdamman** [comment]
or `to_index`?

**Comment by gdementen** [comment]
We already have LArray.indexofmin/max and AxisCollection.index, I would rather keep with the standard .index and not introduce one more term.

**Comment by alixdamman** [comment]
`translate` is only for internal use, right? 
In any case `translate` doesn't speak for itself. 
IMO, it's OK if `Axis.index` returns more than one index in some cases.

**Comment by gdementen** [comment]
> translate is only for internal use, right?

It was indeed meant this way. But I just came on a problem where I had to use it for a user who wanted to know the position of a particular label on a sorted array. Ie what is the rank of 'belgium' in an array sorted by country, and the only way I came up with was:

```python
sorted = array.sort_values('geo')
rank = sorted.geo.translate('belgium')
```

Truth be told, it is really a workaround for the fact we do not have a rank method yet. See #158.
